solver/src/naive_solver/two_order_settlement.rs

```diff
@@ -1,14 +1,55 @@
-use crate::settlement::{Interaction, Settlement, Trade};
+use crate::{
+    interactions::UniswapInteraction,
+    settlement::{Interaction, Settlement, Trade},
+};
 use model::{OrderCreation, OrderKind};
-use primitive_types::U256;
+use primitive_types::{H160, U256};
+use std::collections::HashMap;
+
+#[derive(Debug)]
+pub struct TwoOrderSettlement {
+    clearing_prices: HashMap<H160, U256>,
+    trades: [Trade; 2],
+    interaction: Option<AmmSwapExactTokensForTokens>,
+}
```

Comment:
A new type that only the naive solver uses. This struct is easier to work with locally than the full `Settlement` but can be converted into that.